src/v2/Roboto_Regular.ufo/features.fea

```diff
@@ -939,7 +939,26 @@ feature liga {
     sub f l by f_l;
   } liga02;
 
-  language TUR  exclude_dflt;
+  # include all ligatures for these languages
+  language FRA;  # French
+  lookup liga01;
+  lookup liga02;
+  language MOL;  # Moldavian
+  lookup liga01;
+  lookup liga02;
+  language NAV;  # Navajo
+  lookup liga01;
+  lookup liga02;
+  language ROM;  # Romanian
+  lookup liga01;
+  lookup liga02;
```

Comment:
@jamesgk I'm not sure we need to enumerate both a whitelist and blacklist in this feature. Have you tried something similar to [this snippet](http://blogs.adobe.com/typblography/files/typblography/atypi2006/slye_lisbon-05.pdf#page=98)?